### ISSUE TITLE
tutorial run time

### ISSUE THREAD

**Issue by anna7287** [rstudio/keras#1119]
Hello,

I'm trying to run word2vec using your tutorial (https://blogs.rstudio.com/ai/posts/2017-12-22-word-embeddings-with-keras/) with RStudio (version 4.0.2).

When I run the below command, it never seems to finish (I've let it run overnight). Do you know what I could be doing wrong? I've reduced the epochs, steps per epoch and the fine food text file (by two thirds) to see if training the model would be quicker but to no avail.

>   model %>%
+     fit_generator(
+       skipgrams_generator(reviews, tokenizer, skip_window, negative_samples), 
+       steps_per_epoch = 100, epochs = 1
+     )

Output:
  1/100 [..............................] - ETA: 0s - loss: 0.6931

If you had any advice that would be great.

Thanks!
